etna/metrics/functional_metrics.py

```diff
@@ -41,6 +42,52 @@ def _get_axis_by_multioutput(multioutput: str) -> Optional[int]:
         assert_never(multioutput_enum)
 
 
+def mse_with_missing_handling(y_true: ArrayLike, y_pred: ArrayLike, multioutput: str = "joint") -> ArrayLike:
+    """Mean squared error with missing values handling.
+
+    `Wikipedia entry on the Mean squared error
+    <https://en.wikipedia.org/wiki/Mean_squared_error>`_
```

Comment:
I don't think it is a good idea to reference wikipedia. Better to replace with a more reputable source (e.g. Hyndman Forecasting) or to remove completely.

Reply:
Here I just repeated that we have done with `mape` and `smape`. I could replace link in those places too.